netcode.c

```diff
@@ -6196,11 +6247,30 @@ void test_client_server_connect()
             netcode_server_free_packet( server, packet );
         }
 
+        // Very basic test for checking if each match only has 2 or less clients connected.
```

Comment:
I see this duplicated below - helper method?

Reply:
Super small test that I am removing when I add in the more indepth ones.